sdk/communication/azure-communication-networktraversal/tests/test_communication_relay_client.py

```diff
@@ -35,6 +36,7 @@ def setUp(self):
     def test_get_relay_configuration(self, communication_livetest_dynamic_connection_string):
         identity_client = CommunicationIdentityClient.from_connection_string(
             communication_livetest_dynamic_connection_string,
+            api_version=ApiVersion.V2021_03_07,
```

Comment:
why not the new one here?

Reply:
the same thing as java, instead of regenerating I have just provided GA version

Reply:
right, can you please make a note to remove that later (similarly to java)? you can resolve the conversation then. thanks